WalletWasabi.Fluent/Models/Wallets/WalletSettingsModel.cs

```diff
@@ -101,6 +104,12 @@ private void SetValues()
 		_keyManager.AutoCoinJoin = AutoCoinjoin;
 		_keyManager.PreferPsbtWorkflow = PreferPsbtWorkflow;
 		_keyManager.PlebStopThreshold = PlebStopThreshold;
+		if (ExcludeCoinFromCoinjoinThreshold != _keyManager.ExcludeCoinFromCoinjoinThreshold)
+		{
+			_keyManager.ResetBypassAutomaticExclusionFromCoinjoin();
+			_keyManager.ExcludeCoinFromCoinjoinThreshold = ExcludeCoinFromCoinjoinThreshold;
+			Services.WalletManager.GetWalletByName(_keyManager.WalletName).LoadExcludedCoins();
+		}
```

Comment:
Changing the value for `ExcludeCoinFromCoinjoinThreshold` should invalidate `ypassAutomaticExclusionFromCoinjoin` because it is a specific action from the user